### DISCUSSION TITLE
herd ngrok not working

### DISCUSSION THREAD

**Discussion by danny-cspay** [beyondcode/herd-community#667] [Q&A]
Command `herd share-tool ngrok` is not defined.
Command `herd set-ngrok-token your_token` is not defined.
Command `herd share --hostname=dev.examplesite.com` Error.

<img width="279" alt="image" src="https://github.com/beyondcode/herd-community/assets/137148882/24434a68-379e-4f72-85ca-a3d695a1b826">

<img width="634" alt="image" src="https://github.com/beyondcode/herd-community/assets/137148882/defc9607-3c3d-4466-b837-f54438e24ec9">

**Comment by a-lindsay** [comment]
It seems like ngrok support vanished in v1.6.0, their docs now recommend if you want to use ngrok to use it outside of herd. 

https://herd.laravel.com/docs/1/advanced-usage/sharing-sites#sharing-sites-via-ngrok
